common/src/main/scala/com/gu/emailservices/EmailService.scala

```diff
@@ -32,12 +37,32 @@ case class EmailFields(
        |        "edition": "$edition",
        |        "name": "$name",
        |        "product": "$product"
+       |        $paymentMethodJson
        |      }
        |    }
        |  },
        |  "DataExtensionName": "$dataExtensionName"
        |}
     """.stripMargin
+
+  def paymentMethodJson: String = paymentMethod match {
+    case Some(dd: DirectDebitPaymentMethod) => s"""
+      ,"account name": "${dd.bankTransferAccountName}",
+      "account number": "${mask(dd.bankTransferAccountNumber)}",
```

Comment:
Are we obscuring all the numbers of the account number? If so is it more explicit to put '******'?

Reply:
It's only the first 6. That's what GoCardless use in their example email.